.github/workflows/ci.yml

```diff
@@ -53,20 +53,32 @@ jobs:
 
     container:
       image: ghcr.io/romange/${{ matrix.container }}
+      volumes:
+        - /:/hostroot
+
       credentials:
         username: ${{ github.repository_owner }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           submodules: true
-      - name: Install dependencies
+      - name: Prepare Environment
         run: |
           uname -a
           cmake --version
           mkdir -p ${GITHUB_WORKSPACE}/build
 
+          echo "===================Before freeing up space ============================================"
```

Comment:
Why is this needed?

Reply:
because `-fsanitize` eats up all the space on the runner and we fail building the repo